src/main/java/Alex.java

```diff
@@ -0,0 +1,79 @@
+import java.util.Scanner;
+
+public class Alex {
+    public static void main(String[] args) {
+        Task[] allTasks = new Task[100];
+        int numberOfTasks = 0;
+        Scanner myObj = new Scanner(System.in);  // Create a Scanner object
+        greeting();
+        boolean isBye = false;
+        while(!isBye){
+            String userInput = myObj.nextLine();
+            String[] words = userInput.split(" ");
+            if(words[0].toLowerCase().equals("list")){
+                for(int i = 1; i <= numberOfTasks; i++){
+                    System.out.print(i);
+                    System.out.println(". " + "[" + allTasks[i-1].getStatusIcon() + "]" + " " + allTasks[i-1].getDescription());
+                }
+                printLine();
+
+            }
+            else if(words[0].toLowerCase().equals("bye")){
+                exit();
+                isBye = true;
+                break;
```

Comment:
`isBye = true` is redundant here because `break` will exit from the loop. It may be best if you remove `isBye` from your code as it isn't serving any purpose here.